restapi/src/test/java/io/stargate/web/docsapi/service/query/filter/operation/impl/LtFilterOperationTest.java

```diff
@@ -40,17 +40,17 @@ public void stringEquals() {
     }
 
     @Test
-    public void stringGreater() {
+    public void stringLess() {
       boolean result = lt.test("filterValue", "aaa");
 
-      assertThat(result).isFalse();
+      assertThat(result).isTrue();
```

Comment:
I'm not sure this change correct. `filterValue` is indeed _greater than_ `aaa` lexicographically.

Reply:
Yes that's the point:

* `aaa` is value from the database
* `filterValue` is what's specified from the user (eq. `smth: { lt: 'filterValue'}}
* `aaa` should pass as it's less than `filterValue`

Right?

Reply:
I think I see the problem now. `filterValue` comes from the HTTP request as in `{field: {$lt: "filterValue"}}` as we compare it to "dbValue", right?

IMHO, it would be preferable to reverse the order of parameters to the `test` method, i.e. define it as `test(String dbValue, String filterValue)`. This was the operation will be consistent with `Comparator.compare(dbValue, filterValue))` and with the order of predicate args in the `where` clause at HTTP level. WDYT?

Reply:
Hmm although this is gonna be a simple IDE refactoring, this will create a huge amount of changes. Are we sure we want this? And if it really gives us a better "understanding"?

Reply:
From my POV it will clarity the meaning of those `FilterOperation` classes - closer match with API-level parameters.

A `{field: {$lt: "filterValue"}}` input predicate will translate into a `LtFilterOperation` object. In runtime the `LtFilterOperation.test()` method will be called with values like `test("dbValue", "filterValue")` - it's the same order or parameters as `field` and `filterValue` in the user input and the `true/false` result will be consistent with the contract of `Comparator.test("dbValue", "filterValue")`. WDYT?

Reply:
Yep, I'll add as separate commit.

Reply:
Thanks! Hopefully, in the end, the `isSatisfied()` implementations will not have to change wrt `master`